x-pack/plugins/fleet/server/routes/uninstall_token/handlers.test.ts

```diff
@@ -184,7 +184,8 @@ describe('uninstall token handlers', () => {
     });
   });
 
-  describe('Agent Tamper Protection feature flag', () => {
+  // TODO: remove it when agentTamperProtectionEnabled FF is removed
```

Comment:
why not just remove the test?

Reply:
we plan to remove it once the FF is completely removed. Does that sounds good to you?